cabal.project

```diff
@@ -367,12 +210,48 @@ constraints:
   , Cabal >= 3.4.0.0
   , async-timer >= 0.2.0.0
   , unliftio-core >= 0.2.0.1
+  , cardano-api == 1.35.4
+  , cardano-node == 1.35.4
+  , generic-arbitrary == 0.2.2
+  , iohk-monitoring >= 0.1.11
+
+  -- Could probably be bumped with minor work in the wallet
+  , resource-pool == 0.2.3.2
 
-  -- This pin is just here to make cabalProejct == stackProject.
-  -- Remove it if you are happy for a newer version to be picked when possible.
-  , semigroups == 0.19.2
-  , persistent < 2.13.3.4
+  -- TH Name shadowing warnings need to be addressed when bumping to 2.13.3.5
+  , persistent == 2.13.3.3
 
+  -- Copied from cardano-node's cabal.project:
+  , bimap >= 0.4.0
+  , libsystemd-journal >= 1.4.4
+  , systemd >= 2.3.0
+    -- systemd-2.3.0 requires at least network 3.1.1.0 but it doesn't declare
+    -- that dependency
+  , network >= 3.1.1.0
+  , HSOpenSSL >= 0.11.7.2
+  , algebraic-graphs < 0.7
+  , protolude < 0.3.1
+    -- TODO: these should be set in cabal files, but avoiding setting them in lower dependencies for initial CHaP release
```

Comment:
Lets create another JIRA task and reference it here.
BTW, I don't quite understand the explanation in the comment.

Reply:
🙈 This is a comment copied from cardano-node in the `  -- Copied from cardano-node's cabal.project:` section — I guess I can remove this `TODO` in it.